src/lib/core.js

```diff
@@ -15,14 +15,14 @@ module.exports = function (options) {
     const store = creds.store && newSqliteStore(creds.store)
 
     creds.prefix = ledgerPrefix
-    core.addClient(ledgerPrefix, new ilpCore.Client({
-      plugin: require('ilp-plugin-' + creds.type),
-      auth: creds,
-      store: store,
-      log: makeLogger('plugin-' + creds.type),
+    core.addClient(ledgerPrefix, new ilpCore.Client(Object.assign({
```

Comment:
It seems like giving the built-in options a higher precendence would be wise. `Object.assign({}, creds, {`